Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,44 @@
+
+![pseudo code 2](https://user-images.githubusercontent.com/110966922/202583040-c28e27fe-c175-4287-a8cb-cefafbfe327b.png)
+
+Shown is a schematic of the Arduino Mega 2560 Rev 3 and USB Host Shield:
+![CAPSTONE 1 Microcontroller Schematic Updated](https://user-images.githubusercontent.com/118767661/206057697-6be6d265-4ac2-411e-a632-a95b974fea32.png)
+
+## ANALYSIS OF CHOSEN COMPONENT(S)
+The team has chosen to look at the Arduino Mega 2560 Rev 3 and Arduino USB Shield Host as the solutions.  These components, in conjunction, will satisfy the constraints and jobs of this subsystem.  Following are the corresponding fulfillment of the constraints:
+
+1. The Arduino Mega 2560 Rev 3 falls under the preferred maximum of $60 at a current price of $48.40 (as of 11-8-22).  The USB Host Shield component has a price of $20.99 as of the same date.   
+2. 4 UART serial hardware ports are present on the board.  SPI is supported as well with included libraries.  These enable the Mega to issue instructions and exchange data between other components and subsystems (Communication).
+3. The USB Host Shield enables the use of USB devices such as a Flash drive for external storage and communication. The Arduino Mega does not support USB as a standalone component.
+4. There are 54 digital input/output pins (of which 15 are PWM outputs) and 16 analog inputs.  This meets the needs for at least 18 digital (15 for sensors and 3 for USB Host) and 16 analog input pins.  There are also 4 UART pins and 21 digital (3 for SPI if chosen and 2 for battery monitoring discretely) may be assigned special purposes later on in design.  1 analog pin may also be used for taking a continuous measurement for the battery monitoring.
+5. The code is currently of an unknown size, so it is necessary to aim slightly higher in terms of needed processing memory.  The Arduino Mega has 256 KB of Flash, 8 KB of SRAM, and 4 KB of EEPROM.  The SRAM and Flash memory are satisfactory for the subsystem’s needs.
+6. Algorithmic measurement of the battery capacity is supported.  The board will be able to read the voltage levels coming from the battery by taking a reading from a voltage divider located in the power subsystem.  This reading can then be used as a reference initially and then measured against the incoming voltage from said point continuously/discretely. Running an algorithm based on the battery’s expected output vs operational output will determine the battery’s capacity.
```

Comment:
The details should be part of the constraint. This just shows how the device meets the constraint. (But modify all of the above per other conversations about moving battery monitoring to the power subsystem.)